src/main/resources/reference.conf

```diff
@@ -224,6 +224,41 @@ resourceTypes = {
     }
     reuseIds = false
   }
+  google-project = {
+    actionPatternObjects = {
+      list_notebook_cluster = {
+        description = "list all notebook clusters in this billing-project"
+      }
+      launch_notebook_cluster = {
+        description = "launch a new notebook cluster in this billing-project"
+      }
+      delete_notebook_cluster = {
+        description = ""
+      }
+      stop_start_notebook_cluster = {
+        description = "stop and start notebook clusters in this billing-project"
+      }
+      list_persistent_disk = {
+        description = "list all persistent disks in this billing-project"
+      }
+      create_persistent_disk = {
+        description = "create persistent disk in this billing-project"
+      }
+      delete_persistent_disk = {
+        description = "delete persistent disk in this billing-project"
+      }
+      create_kubernetes_app = {
+        description = "create kubernetes application in this billing-project"
+      }
+    }
+    ownerRoleName = "owner"
+    roles = {
+      owner = {
+        roleActions = ["list_notebook_cluster", "launch_notebook_cluster", "delete_notebook_cluster", "list_persistent_disk", "create_persistent_disk", "delete_persistent_disk", "create_kubernetes_app"]
```

Comment:
There are a few non-Leo compute actions that should probably be added to this list: `alter_policies`, `read_policies`, `add_to_service_perimeter`, and `delete`.

One could make the case for `alter_google_role` to be added here as well, but I think it's not really used anymore after Doug's recent Rawls PR

Reply:
yeah, drop alter_google_role
add_to_service_perimeter should not be added, that is remaining at the billing-project level